lib/ansible/modules/cloud/ovirt/ovirt_nics.py

```diff
@@ -170,7 +188,8 @@ def main():
             choices=['present', 'absent', 'plugged', 'unplugged'],
             default='present'
         ),
-        vm=dict(required=True),
+        vm=dict(required=False),
```

Comment:
`required=False` is default, no need to specify.
Please use `required_if`: https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/ovirt/ovirt_auth.py#L198